EXAMPLES.md

```diff
@@ -129,6 +130,23 @@ const { data: tokens } = await auth.passwordless.loginWithEmail({
 });
 ```
 
+### mTLS request
+
+```js
+import { AuthenticationClient } from 'auth0';
+
+const auth = new AuthenticationClient({
+  domain: '{YOUR_TENANT_AND REGION}.auth0.com',
```

Comment:
How is `mtls` added to the domain?

Reply:
If customers want to use mTLS, they have to configure the domain value with a `mtls.` prefix on whatever domain they're using (custom or canonical).

@gyaneshgouraw-okta perhaps that part can be made more clear here in the docs. Consider also linking to the Auth0 docs for the developer to go and read more: https://auth0.com/docs/get-started/authentication-and-authorization-flow/authenticate-with-mtls

Reply:
I am confused reading the docs.

> Environments that support mTLS endpoint aliases expose a new property, mtls_endpoint_aliases, that contains a list of endpoints that support mTLS. For clients that support mTLS, the endpoints listed under mtls_endpoint_aliases take precedence over the same endpoints exposed outside of mtls_endpoint_aliases.

> If an endpoint is not listed under mtls_endpoint_aliases, use the same endpoint listed outside of mtls_endpoint_aliases. In the example above, pushed_authorization_request_endpoint is not listed under mtls_endpoint_aliases. As a result, use the pushed_authorization_request_endpoint exposed outside of mtls_endpoint_aliases, or https://auth.bank.com/oauth/par.

This reads as if not all endpoints support mtls, how can customers distinguish this when they have to add the `mtls` prefix globally?

Reply:
Tested the well-known endpoint, and afaict it does not work when called with the `mtls` prefix.

Reply:
Updated the changes as discussed, made use of new param `useMTLS`. Which gives more control over domain name and it's prefixing with `mTLS` for different authentication endpoints